.github/workflows/deploy-previews.yml

```
@@ -1,879 +1,879 @@
-    steps:
-      - uses: actions/github-script@v6
-        with:
-          script: |
-            github.rest.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: 'website changes in this PR available for preview [here](${{ needs.website.outputs.url }})'
-            })           
\ No newline at end of file
+# name: deploy-to-preview-environment
```

actionlint (1.7.12)

[error] workflow is empty

(syntax-check)